corehq/apps/userreports/models.py

```diff
@@ -95,6 +95,12 @@ class Meta(object):
         # prevent JsonObject from auto-converting dates etc.
         string_conversions = ()
 
+    @classmethod
+    def wrap(self, data):
+        if not data.get('backend_id', None):
```

Comment:
Not in love with this change, but it's a workaround to get the test decorator to work